amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp

```diff
@@ -221,7 +264,6 @@ static vx_status VX_CALLBACK initializeNormalize(vx_node node, const vx_referenc
         data->pDstGenericDesc->offsetInBytes = 0;
         fillGenericDescriptionPtrfromDims(data->pDstGenericDesc, data->inputLayout, data->outputTensorDims);
 
-        data->pSrcDims = new uint[data->inputTensorDims[0] * 2];
         refreshNormalize(node, parameters, num, data);
```

Comment:
Why this is removed we can do it here right?

Reply:
Normalize node only supported Host kernel so they created host buffer here. Since now we are adding normalize HIP, I removed this and created host/pinned memory in refresh function